src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -30,18 +31,21 @@ public async Task PullImageAsync(string outputFolder)
             await OrasExecutionAsync(command, Directory.GetCurrentDirectory());
         }
 
-        public async Task PushImageAsync(string inputFolder)
+        public async Task PushImageAsync(string inputFolder, List<string> filePathList)
```

Comment:
Can we use a single argument instead of parsing folder and filenameList?

Reply:
Seems not easy to separate working folder and file name which combined in parameter.

Reply:
it's a hot-fix PR. we can discuss it later.